include/AudioPortAudio.h

```diff
@@ -151,8 +147,8 @@ class AudioPortAudio : public AudioDevice
 	bool m_wasPAInitError;
 
 	SampleFrame* m_outBuf;
-	int m_outBufPos;
-	int m_outBufSize;
+	std::size_t m_outBufPos;
```

Comment:
Do we really need that std:: there?

Reply:
It'll work without it under the major compilers, but the C++ standard doesn't guarantee that `size_t` will be available in the global namespace. See <https://timsong-cpp.github.io/cppwp/n4659/headers#4>:
> It is unspecified whether these names [...] are first declared within the global namespace scope and are then injected into namespace `std` by explicit *using-declaration*s.